osquery/tables/system/hash.cpp

```diff
@@ -182,6 +186,21 @@ bool FileHashCache::load(const std::string& path, MultiHashes& out) {
   return true;
 }
 
+std::string genSsdeepForFile(const std::string& path) {
+#ifndef WINDOWS
+  std::string file_ssdeep_hash(FUZZY_MAX_RESULT, '\0');
+  auto did_ssdeep_fail =
+      fuzzy_hash_filename(path.c_str(), &file_ssdeep_hash.front());
+  if (did_ssdeep_fail) {
+    LOG(WARNING) << "ssdeep failed: " << path;
+    return "-1";
```

Comment:
As far as `"-1"` is a default value for this column, it it better to define it as a named constant and use constant by name in such places. What do you think?